app/build.gradle

```diff
@@ -12,7 +12,6 @@ android {
     }
     kotlinOptions {
         jvmTarget = '1.8'
-        useIR = true
```

Comment:
Why Remove this?

Reply:
It was showing error on gradle, let me undo and post the error if i got any